### PR TITLE
Maintenance: Clean up some stuff around settings

### PR DESCRIPTION
**Pull request by turt2live**
Fixes https://github.com/vector-im/riot-web/issues/8864
Removes need for `DragDropContext` over flair settings
Fixes a technical bug where one could drag groups from settings over the LLP
Part of https://github.com/vector-im/riot-web/issues/8207

*Note*: It might be easiest to review this commit-by-commit

### CODE REVIEW

File: res/css/_components.scss [hunk 1]
```diff
@@ -150,16 +150,16 @@
...
-@import "./views/settings/tabs/_HelpSettingsTab.scss";
-@import "./views/settings/tabs/_NotificationSettingsTab.scss";
-@import "./views/settings/tabs/_PreferencesSettingsTab.scss";
-@import "./views/settings/tabs/_RolesRoomSettingsTab.scss";
-@import "./views/settings/tabs/_SecurityRoomSettingsTab.scss";
-@import "./views/settings/tabs/_SecuritySettingsTab.scss";
 @import "./views/settings/tabs/_SettingsTab.scss";
-@import "./views/settings/tabs/_VoiceSettingsTab.scss";
+@import "./views/settings/tabs/room/_GeneralRoomSettingsTab.scss";
+@import "./views/settings/tabs/room/_RolesRoomSettingsTab.scss";
```
Comment (bwindels) [inline res/css/_components.scss]: nit: IMO `settings/room/tabs` and `settings/user/tabs` makes more sense but this works too.

Comment (turt2live) [reply res/css/_components.scss]: That sounds like a good improvement to make if we ever get rid of `room_settings` being a thing elsewhere (which I also want to do, but not right now)

Comment (turt2live) [reply res/css/_components.scss]: Have opened https://github.com/vector-im/riot-web/issues/8934 to track this

**Comment by turt2live** [comment]
Merging because I'm confident the build will pass and I'm losing patience with waiting 2 hours for a build.

**Pull request merged**